data/json/items/tool/toileteries.json

```diff
@@ -230,9 +230,9 @@
     "type": "TOOL",
     "name": { "str": "washboard" },
     "description": "This is a wooden washboard.  You can use it to wash filthy clothing if it's supplied with cleansing agent.",
-    "weight": "786 g",
-    "volume": "1966 ml",
-    "longest_side": "40 cm",
+    "weight": "116 kg",
```

Comment:
I don't think this weight is correct.

Reply:
> I don't think this weight is correct.
Oh yes, confusion when i converted back from G to KG i think. It is 1.16KG. Agh. thought it was ready to go, apparently not. I'll take anther look.